src/screens/Books/index.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+import { connect } from 'react-redux';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+
+import { actionCreators as bookActions } from '../../redux/books/actions';
+import { Spinner } from '../../components/Spinner';
+import { ErrorModal } from '../../components/ErrorModal';
+import TextButton from '../../components/TextButton';
+import List from '../../components/List';
+
+import ListItem from './components/ListItem';
+import styles from './styles';
+
+class Books extends Component {
+  async componentDidMount() {
+    this.loadContent();
```

Comment:
I think you should either return `this.loadContet()` or either await for it.
If you don't, `this.loadContent` generates a promise that you are not waiting for it to finish.

Reply:
To solve this, I removed async from `componentDidMount()` as `loadContent()` is an async function.